README.md

```diff
@@ -58,7 +57,12 @@ The models are fully compatible with the latest release, they can be used for te
 
 Documentation, including JavaDocs, code usage and command-line interface examples are available [here](http://opennlp.apache.org/docs/)
 
-You can also follow our [mailing lists](http://opennlp.apache.org/mailing-lists.html) for news and updates, or via social media channels: [![Twitter Follow](https://img.shields.io/twitter/follow/ApacheOpenNLP.svg?style=social)](https://twitter.com/ApacheOpenNLP)
+For recent news, updates and topics, you can follow  
+- the regular [mailing lists](http://opennlp.apache.org/mailing-lists.html), 
+- the [![Slack](https://img.shields.io/badge/Slack-4A154B?logo=slack&logoColor=fff)](https://the-asf.slack.com) channel, or 
```

Comment:
Might need a note, that Slack is only open for people with @apache.org mails (or upon explicit invitation)

Reply:
suggest sth. pls

Reply:
"the [![Slack](https://img.shields.io/badge/Slack-4A154B?logo=slack&logoColor=fff)](https://the-asf.slack.com) channel (only available to people with `@apache.org` email addresses or upon invitation)"